CHANGES.current

```diff
@@ -7,6 +7,17 @@ the issue number to the end of the URL: https://github.com/swig/swig/issues/
 Version 4.2.0 (in progress)
 ===========================
 
+2023-XX-XX: degasus
+            Fix integer overflow / undefined behavior in the python cast_mode on sizeof(long)==8
+            plattforms for the implicit convertion of the edge cases:
```

Comment:
Typo: "plattforms"

Reply:
Also "convertion" (should be "conversion").

And the convention is to "tag" the entry with the language it applies to like so:

```
2023-12-01: saiarcot895
            [Python] #2413 Prevent potential multi-threading crash; gracefully exit running
            daemon threads on main thread exit.
```

Reply:
Done.
Nitpicking: Which convention should I follow for the indentation? 12 spaces or one tab + 4 spaces?

And as usual, sorry for my bad English ;-)

Reply:
SWIG coding style expects a tab to move to the next multiple of 8 column so they're equivalent.